### ISSUE TITLE
Missing event time in HTTP API response

### ISSUE THREAD

**Issue by enumag** [EventStore/EventStore#1863]
I wrote some events to eventstore and added a projection to aggregate events from multiple streams to one stream. Now I'm reading those events using HTTP API and receive a response like this:

```
{
  "title": "Event stream 'everything'",
  "id": "http://127.0.0.1:2113/streams/everything",
  "updated": "2019-02-14T07:58:56.074568Z",
  "streamId": "everything",
  "author": {
    "name": "EventStore"
  },
  "headOfStream": false,
  "selfUrl": "http://127.0.0.1:2113/streams/everything",
  "links": [
    {
      "uri": "http://127.0.0.1:2113/streams/everything",
      "relation": "self"
    },
    {
      "uri": "http://127.0.0.1:2113/streams/everything/head/backward/20",
      "relation": "first"
    },
    {
      "uri": "http://127.0.0.1:2113/streams/everything/20/forward/20",
      "relation": "previous"
    },
    {
      "uri": "http://127.0.0.1:2113/streams/everything/metadata",
      "relation": "metadata"
    }
  ],
  "entries": [
    {
      "eventId": "dd8af741-a573-894f-8a46-945a4ca5d364",
      "eventType": "ContactAdministration.ContactCard.ContactCardRegistered",
      "eventNumber": 0,
      "data": "{\n  \"contactCardId\": \"fd882842-dfbb-0d4b-ac55-4d30284d4999\",\n  \"details\": {\n    \"viewName\": \"Kateřina Svobodová\",\n    \"note\": null,\n    \"personalInformation\": {\n      \"salutation\": \"mrs\",\n      \"firstName\": \"Kateřina\",\n      \"lastName\": \"Svobodová\",\n      \"titleBefore\": null,\n      \"titleAfter\": null\n    },\n    \"businessInformation\": null\n  }\n}",
      "metaData": "{\n  \"_eventType\": \"ContactAdministration.ContactCard.ContactCardRegistered\",\n  \"_eventId\": \"dd8af741-a573-894f-8a46-945a4ca5d364\",\n  \"_createdAt\": \"2019-02-14T07:58:55+00:00\",\n  \"_aggregateType\": \"App\\\\Module\\\\ContactAdministration\\\\Domain\\\\ContactCard\\\\ContactCard\",\n  \"_aggregateId\": \"fd882842-dfbb-0d4b-ac55-4d30284d4999\",\n  \"_version\": -1\n}",
      "linkMetaData": "{\n  \"$v\": \"5:-1:0:3\",\n  \"$s\": {\n    \"$ce-Accounting.AccountingOrganization\": -1,\n    \"$ce-Accounting.FinancialAccountGroup\": -1,\n    \"$ce-CodeListsManagement.UnitAttributeSet\": 1,\n    \"$ce-CodeListsManagement.UnitCategory\": -1,\n    \"$ce-CodeListsManagement.FeeName\": 4,\n    \"$ce-CodeListsManagement.ZipCode\": 1,\n    \"$ce-CodeListsManagement.UnitAttribute\": 1,\n    \"$ce-ContactAdministration.ContactCard\": 8,\n    \"$ce-FeeCalculating.MonthlyAuditProcess\": -1,\n    \"$ce-FeeCalculating.RealtyAuditor\": -1,\n    \"$ce-FeeCalculating.FeeTemplate\": -1,\n    \"$ce-FeeItemRegistry.FeeItem\": -1,\n    \"$ce-PaymentAllocating.AllocationContext\": -1,\n    \"$ce-PaymentAllocating.PaymentAllocationsProcess\": -1,\n    \"$ce-PaymentAllocating.PaymentAllocationsAuditor\": -1,\n    \"$ce-PaymentImport.PaymentImportAuditor\": -1,\n    \"$ce-PaymentRegistry.Payment\": -1,\n    \"$ce-RealtyRegistration.Unit\": -1,\n    \"$ce-RealtyRegistration.Entrance\": -1,\n    \"$ce-RealtyRegistration.Realty\": -1,\n    \"$ce-RealtyRegistration.Contract\": -1\n  },\n  \"$causedBy\": \"dd8af741-a573-894f-8a46-945a4ca5d364\"\n}",
      "streamId": "ContactAdministration.ContactCard-fd882842-dfbb-0d4b-ac55-4d30284d4999",
      "isJson": true,
      "isMetaData": true,
      "isLinkMetaData": true,
      "positionEventNumber": 19,
      "positionStreamId": "everything",
      "title": "0@ContactAdministration.ContactCard-fd882842-dfbb-0d4b-ac55-4d30284d4999",
      "id": "http://127.0.0.1:2113/streams/ContactAdministration.ContactCard-fd882842-dfbb-0d4b-ac55-4d30284d4999/0",
      "updated": "2019-02-14T07:59:26.83966Z",
      "author": {
        "name": "EventStore"
      },
      "summary": "$>",
      "links": [
        {
          "uri": "http://127.0.0.1:2113/streams/ContactAdministration.ContactCard-fd882842-dfbb-0d4b-ac55-4d30284d4999/0",
          "relation": "edit"
        },
        {
          "uri": "http://127.0.0.1:2113/streams/ContactAdministration.ContactCard-fd882842-dfbb-0d4b-ac55-4d30284d4999/0",
          "relation": "alternate"
        }
      ]
    },
...
```

As you can see the response only contains one time field `"updated": "2019-02-14T07:59:26.83966Z",`.

It is unclear whether this is the time of the "link event" contained within the projection stream or the time of the "linked event" contained in one of the aggregated streams.

Either way the time of the other event is missing here. I'd like to know the times of both the real event and the link event since they are not the same.

**Comment by gregoryyoung** [comment]
I believe the time is off the link event or the resolved event depending on if it is resolved. 

Adding both times should not be an issue, I have made an issue to track. https://github.com/EventStore/EventStore/issues/1865

**Comment by mat-mcloughlin** [comment]
closed as it is being tracked via #1865
